Fix expires_on check for verisign domains

phcyso opened the pull request:
It appears verisign has updated the wording for the expiry date for .com domains
Confimed with: 
```
# whois -h whois.verisign-grs.com google.com
   Domain Name: GOOGLE.COM
...
   Registry Expiry Date: 2020-09-14T04:00:00Z
...

```

weppos commented:
The change is breaking the build, and also it doesn't include appropriate test updates and fixtures. Please provide the appropriate test updates as well.

phcyso commented:
@weppos Sorry about that, i didn't think to check the specs.
Have updated the rest of the changed fields and updated the fixtures and specs to match.

I am not sure how the `./spec/fixtures/responses/whois.verisign-grs.com/com/property_registrar_with_multiple_entries.txt` was originally created as my whois clients on osx and linux only return one record now so i have updated this file to match the new field name manually.

weppos commented:
Thanks @phcyso, that's a great help!

> I am not sure how the ./spec/fixtures/responses/whois.verisign-grs.com/com/property_registrar_with_multiple_entries.txt was originally created as my whois clients on osx and linux only return one record now so i have updated this file to match the new field name manually.

I was unable to reproduce it. It looks like Verisign finally decided to turn the default to an exact match search:
https://github.com/weppos/whois/pull/559/commits/b28c9ed0ee03e0575b2bc645a963b360fc03a36d

I'll remove the fixture and the test.

weppos commented:
@phcyso how did you generate the fixtures? Did you use the [`mkwhois`](https://github.com/weppos/whois-parser/blob/master/utils/mkwhois.rb) tool? It looks like the fixtures you added contains both the verisign and the referral response, whereas they should only include the one from verisign.

weppos commented:
Actually, I may be wrong, the fixtures seem correct.
